packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.h

```diff
@@ -7,7 +7,7 @@
 @interface FLTImagePickerPlugin : NSObject <FlutterPlugin>
 
 // For testing only.
```

Comment:
Should we remove this line now?

Reply:
I don't think so? Both of these methods here are exposed only for testing and not meant to be used in normal circumstances.